src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -82,8 +83,9 @@
     <link rel="stylesheet" href="css/jquery.mCustomScrollbar.min.css">
     <link rel="stylesheet" href="css/pageguide.min.css"/>
     <link rel="stylesheet" href="css/app.css">
-    <link rel="stylesheet" href="css/jquery.dropdown.css"/>
+    <link rel="stylesheet" href="css/main.css">
     <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:500,900" rel="stylesheet">
```

Comment:
Thanks. Fixed in 9f62d89.

![image](https://cloud.githubusercontent.com/assets/1042475/19449512/d587ab3e-9473-11e6-91c4-4d6215f24fcd.png)